libavfilter/vf_livepeer_dnn.c

```diff
@@ -36,80 +36,45 @@
+    struct AVFrame *swscaleframe;  ///< Scaled image
+    FILE *logfile;       ///< (Optional) Log classification probabilities in this file
+    char *log_filename;  ///< File name
 } LivepeerContext;
 
 #define OFFSET(x) offsetof(LivepeerContext, x)
 #define FLAGS AV_OPT_FLAG_FILTERING_PARAM | AV_OPT_FLAG_VIDEO_PARAM
 static const AVOption livepeer_options[] = {
-    { "dnn_backend", "DNN backend used for model execution", OFFSET(dnnctx.backend_type), AV_OPT_TYPE_INT, { .i64 = 1 }, 0, 1, FLAGS, "backend" },
-    { "native", "native backend flag", 0, AV_OPT_TYPE_CONST, { .i64 = 0 }, 0, 0, FLAGS, "backend" },
+        {"dnn_backend", "DNN backend used for model execution", OFFSET(dnnctx.backend_type), AV_OPT_TYPE_INT,
+         {.i64 = 1}, 0, 1, FLAGS, "backend"},
+        {"native", "native backend flag", 0, AV_OPT_TYPE_CONST, {.i64 = 0}, 0, 0, FLAGS, "backend"},
 #if (CONFIG_LIBTENSORFLOW == 1)
-    { "tensorflow", "tensorflow backend flag", 0, AV_OPT_TYPE_CONST, { .i64 = 1 }, 0, 0, FLAGS, "backend" },
+        {"tensorflow", "tensorflow backend flag", 0, AV_OPT_TYPE_CONST, {.i64 = 1}, 0, 0, FLAGS, "backend"},
 #endif
-    { "model", "path to model file specifying network architecture and its parameters", OFFSET(dnnctx.model_filename), AV_OPT_TYPE_STRING, {.str=NULL}, 0, 0, FLAGS },
-    { "input",       "input name of the model",     OFFSET(dnnctx.model_inputname),  AV_OPT_TYPE_STRING,    { .str = "x" },  0, 0, FLAGS },
-    { "output",      "output name of the model",    OFFSET(dnnctx.model_outputname), AV_OPT_TYPE_STRING,    { .str = "y" },  0, 0, FLAGS },
-    // default session_config = {allow_growth: true}
-    { "backend_configs",     "backend configs",     OFFSET(dnnctx.backend_options),  AV_OPT_TYPE_STRING,    { .str = "sess_config=0x01200232" }, 0, 0, FLAGS },
-    { "logfile", "path to logfile", OFFSET(log_filename), AV_OPT_TYPE_STRING, { .str = NULL }, 0, 0, FLAGS },
-    { NULL }
+        {"model", "path to model file specifying network architecture and its parameters",
+         OFFSET(dnnctx.model_filename), AV_OPT_TYPE_STRING, {.str=NULL}, 0, 0, FLAGS},
+        {"input", "input name of the model", OFFSET(dnnctx.model_inputname), AV_OPT_TYPE_STRING, {.str = "x"}, 0, 0,
+         FLAGS},
+        {"output", "output name of the model", OFFSET(dnnctx.model_outputname), AV_OPT_TYPE_STRING, {.str = "y"}, 0, 0,
+         FLAGS},
+        // default session_config = {allow_growth: true}
+        {"backend_configs", "backend configs", OFFSET(dnnctx.backend_options), AV_OPT_TYPE_STRING,
+         {.str = "sess_config=0x01200232"}, 0, 0, FLAGS},
+        {"logfile", "path to logfile", OFFSET(log_filename), AV_OPT_TYPE_STRING, {.str = NULL}, 0, 0, FLAGS},
+        {NULL}
 };
 
 AVFILTER_DEFINE_CLASS(livepeer);
 
 static int post_proc(AVFrame *out, DNNData *dnn_output, AVFilterContext *context);
 
-static int pre_exec(AVFilterContext *context)
-{
+static av_cold int init(AVFilterContext *context) {
```

Comment:
Most of the formatting changes and fixes done by your IDE here are OK.

With a few exceptions that are more or less consistent throughout the FFmpeg codebase and we should stick to -

1. Opening brace `{` comes on a new line for function definitions
2. Indent-width of struct members is 4 spaces (instead of 8 spaces used in this PR)

Reverting these manually might be a pain though - maybe your IDE supports configuring these?